watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/swine_wwt/swine_wwt.py

```diff
@@ -0,0 +1,711 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+import os
+from pyomo.environ import (
+    ConcreteModel,
+    Set,
+    Expression,
+    value,
+    TransformationFactory,
+    units as pyunits,
+    assert_optimal_termination,
+    Block,
+)
+from pyomo.network import Arc, SequentialDecomposition
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.solvers import get_solver
+from idaes.models.unit_models import Product
+import idaes.core.util.scaling as iscale
+from idaes.core import UnitModelCostingBlock
+
+from watertap.core.util.initialization import assert_degrees_of_freedom
+
+from watertap.core.wt_database import Database
+import watertap.core.zero_order_properties as prop_ZO
+from watertap.unit_models.zero_order import (
+    FeedZO,
+    AnaerobicMBRMECZO,
+    CofermentationZO,
+    ConstructedWetlandsZO,
+    GasSpargedMembraneZO,
+    IonExchangeZO,
+    SedimentationZO,
+    VFARecoveryZO,
+)
+from idaes.models.unit_models import Mixer, MomentumMixingType, MixingType
+from watertap.core.zero_order_costing import ZeroOrderCosting
+from watertap.costing import WaterTAPCosting
+
+
+def main():
+    m = build()
+
+    set_operating_conditions(m)
+    assert_degrees_of_freedom(m, 0)
+    assert_units_consistent(m)
+
+    initialize_system(m)
+
+    results = solve(m)
+    assert_optimal_termination(results)
```

Comment:
Should we be adding a `try/except` on this (or perhaps doing a `check_optimal_termination` instead of `assert_optimal_termination`)? In the current form, if the result in not optimal the user will see an `AssertionError` with  default message which might not be that useful for explaining what happened.

Reply:
@andrewlee94 I think replacing with `check_optimal_termination` in all AMO flowsheets in a separate PR would be good. Any other issues that need to be addressed now before you approve?